docs/docs/reference/release-notes/v5.13/index.md

```diff
@@ -0,0 +1,80 @@
+---
+date: "2023-12-@TODO"
+version: "5.13.0"
+title: "v5.13 Release Notes"
+---
+
+Welcome to `gatsby@5.13.0` release (December 2023 #1)
+
+Key highlight of this release:
+
+- [Custom Image and File CDN URL generators](#custom-image-and-file-cdn-url-generators)
+
+Also check out [notable bugfixes](#notable-bugfixes--improvements).
+
+**Bleeding Edge:** Want to try new features as soon as possible? Install `gatsby@next` and let us know if you have any [issues](https://github.com/gatsbyjs/gatsby/issues).
+
+[Previous release notes](/docs/reference/release-notes/v5.12)
+
+[Full changelog][full-changelog]
+
+---
+
+## Custom Image and File CDN URL generators
+
+[`gatsby@4.10.0` introduced Image CDN](/docs/reference/release-notes/v4.10/#image-cdn) capability for Gatsby users. For best build performance it requires deployed site to handle image transformations at request time instead of at build time. Until now, handling those requests required supporting Gatsby specific URL format. With this release we're introducing support for custom URL generators that allow you to use your own URL format making it easier to integrate with existing site deployment platforms that offer Image CDN capabilities or using self managed Image CDNs by providing a way to generate URLs that are already in format expected by your CDN.
+
+Adapter API now offers ability to provide custom URL generators modules for both Image CDN (`gatsbyImage`) and File CDN (`publicUrl`). `gatsby-adapter-netlify@1.1.0` is optioanlly using this feature to make use of [Netlify's Image CDN](https://docs.netlify.com/image-cdn/overview/). Be sure to check [Netlify Adapter documentation](https://www.npmjs.com/package/gatsby-adapter-netlify#imagecdn) for more information how to enable and configure it.
```

Comment:
adapter npm readme link is not yet because it wasn't released to stable YET, but https://www.npmjs.com/package/gatsby-adapter-netlify/v/next#imagecdn (readme for `@next`) shows what it would link to once released to stable